### DISCUSSION TITLE
[Project Proposal Review]Chatbot for AsyncAPI creators

### DISCUSSION THREAD

**Discussion by HARSHJHUNJHUNWALABits** [postmanlabs/gsoc#42] [Ideas]
Dear Mentors,
I am Harsh Jhunjhunwala, a third-year B.E Computer Science student at BITS Pilani.
I have proposed IDEA-12 under the title "CHATBOT FOR ASYNC API CREATORS". I will be highly grateful if mentors could give their feedback and suggest edits.
I am excited to contribute to the idea and work on it.
Thanking You,
HARSH JHUNJHUNWALA.
Here is a link to the google doc,
https://docs.google.com/document/d/1ph0F-M-A2NfPWFKeOUzqO4Vuejs2JwBdko1EgVd8ZMc/edit?usp=sharing

**Comment by Sharath-Postman** [comment]
Thank you so much for writing in. The mentors have been notified to review your proposal.

Cheers
